update!: Move Molly from blog post to RTC page (#2729)
privacyguides/privacyguides.org: docs/real-time-communication.md

![Molly logo](assets/img/messengers/molly.svg){ align=right }

**Molly** is an alternative app that uses the Signal network and offers additional security features, including locking the app at rest, securely shredding unused RAM data, routing via Tor, and [more](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening#privacy-and-security-features). If your threat model requires protecting against [:material-target-account: Targeted Attacks](basics/common-threats.md#attacks-against-specific-individuals){ .pg-red }, then Molly's extensive hardening and anti-forensic features could prove to be useful.

[:octicons-home-16: Homepage](https://molly.im){ .md-button .md-button--primary }
[:octicons-eye-16:](https://signal.org/legal/#privacy-policy){ .card-link title="Privacy Policy" }
[:octicons-info-16:](https://github.com/mollyim/mollyim-android/wiki){ .card-link title="Documentation"}
[:octicons-code-16:](https://github.com/mollyim/mollyim-android){ .card-link title="Source Code" }
[:octicons-heart-16:](https://opencollective.com/mollyim){ .card-link title="Contribute" }

<details class="downloads" markdown>
<summary>Downloads</summary>

> - [:octicons-moon-16: Accrescent](https://accrescent.app/app/im.molly.app)
> - [:simple-github: GitHub](https://github.com/mollyim/mollyim-android/releases)

Reviewer: ```suggestion
- [:simple-fdroid: F-Droid](https://molly.im/fdroid)
- [:octicons-moon-16: Accrescent](https://accrescent.app/app/im.molly.app)
- [:simple-github: GitHub](https://github.com/mollyim/mollyim-android/releases)
```

Author: I thought there was an unspoken rule to not include F-Droid download links unless the software is only distributed through there (a prominent example being Mull).

Reviewer: I believe it can/should be linked when they are using a developer-operated repository.

Regardless, I don't think Accrescent can be the *only* app store download for a listing when the app also recommends other app stores (F-Droid in this case), given that we intentionally exclude Accrescent from https://www.privacyguides.org/en/android/obtaining-apps/.

[Signal Configuration and Hardening :material-arrow-right-drop-circle:](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening)

#### Molly (Android)

> 
> <div class="admonition recommendation" markdown>
> 
> ![Molly logo](assets/img/messengers/molly.svg){ align=right }
> 
> **Molly** is an alternative app that uses the Signal network and offers additional security features, including locking the app at rest, securely shredding unused RAM data, routing via Tor, and [more](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening#privacy-and-security-features). If your threat model requires protecting against [:material-target-account: Targeted Attacks](basics/common-threats.md#attacks-against-specific-individuals){ .pg-red }, then Molly's extensive hardening and anti-forensic features could prove to be useful.

Reviewer: I feel like for people who don't pay attention to heading levels it is not _immediately_ clear that this is an alternative client and not a separate service. Maybe an introduction along these lines?

```suggestion

If you use Android and your threat model requires protecting against [:material-target-account: Targeted Attacks](basics/common-threats.md#attacks-against-specific-individuals){ .pg-red  } you may consider using this alternative app, which features a number of security and usability improvements, to access the Signal network.

<div class="admonition recommendation" markdown>

![Molly logo](assets/img/messengers/molly.svg){ align=right }

**Molly** is an alternative Signal client for Android which allows you to encrypt the local database with a passphrase at rest, to have unused RAM data securely shredded, to route your connection via Tor, and [more](https://blog.privacyguides.org/2022/07/07/signal-configuration-and-hardening#privacy-and-security-features). It also has usability improvements including scheduled backups, automatic locking, and the ability to use your Android phone as a linked device instead of the primary device for a Signal account.
```

[:octicons-home-16: Homepage](https://molly.im){ .md-button .md-button--primary }
[:octicons-eye-16:](https://signal.org/legal/#privacy-policy){ .card-link title="Privacy Policy" }
[:octicons-info-16:](https://github.com/mollyim/mollyim-android/wiki){ .card-link title="Documentation"}
[:octicons-code-16:](https://github.com/mollyim/mollyim-android){ .card-link title="Source Code" }
[:octicons-heart-16:](https://opencollective.com/mollyim){ .card-link title="Contribute" }

<details class="downloads" markdown>
<summary>Downloads</summary>

- [:simple-fdroid: F-Droid](https://molly.im/fdroid)
- [:octicons-moon-16: Accrescent](https://accrescent.app/app/im.molly.app)
- [:simple-github: GitHub](https://github.com/mollyim/mollyim-android/releases)

</details>

</div>

Molly is updated every two weeks to include the latest features and bug fixes from Signal. The exception is security issues, which are patched as soon as possible. That said, you should be aware that there might be a slight delay compared to upstream, which may affect actions such as [migrating from Signal to Molly](https://github.com/mollyim/mollyim-android/wiki/Migrating-From-Signal#migrating-from-signal).

> Note that by using Molly, you are extending your trust to another party, as you now need to trust the Molly team as well as the Signal team.

Reviewer: is this comma necessary? idk looks off to me

```suggestion
Note that by using Molly you are extending your trust to another party, as you now need to trust the Molly team as well as the Signal team.
```

could also be:

```suggestion
Note that you are trusting multiple parties by using Molly, as you now need to trust the Signal team *and* the Molly team to deliver safe updates.
```

Author: :+1:  I like the emphasis on "and" in your second suggestion.

> 

Reviewer: I think we could probably mention this on this page as well:

```suggestion

There is a version of Molly called **Molly-FOSS** which removes proprietary code like the Google services used by both Signal and Molly, at the expense of some features like push notifications. There is also a version called [**Molly-UP**](https://github.com/mollyim/mollyim-android/tree/main?tab=readme-ov-file#unifiedpush) which is based on Molly-FOSS and adds back support for push notifications with UnifiedPush, but it requires self-hosting a program on a separate computer to function. All three versions of Molly provide the same security improvements.

```

(not linking to Molly-FOSS because the download links are the same as Molly's)

Author: Good idea. I even think that the line about reproducible builds can be moved here.

I was hesitant to do that before due to the length of the listing, but since Molly's occupying a subheading, I guess that's not too much of a concern.
